apps/test-suite/tests/FileSystemNext.ts

```diff
@@ -150,13 +150,43 @@ export async function test({ describe, expect, it, ...t }) {
         expect(folder.exists).toBe(true);
       });
 
+      it('Creates a folder with subdirectories if intermediates is set', () => {
+        const folder = new Directory(testDirectory, 'some', 'subdirectory', 'here');
+        folder.create({ intermediates: true });
+        expect(folder.exists).toBe(true);
+      });
+
+      it('Throws en error while creating a folder with subdirectories if intermediates is not set', () => {
+        const folder = new Directory(testDirectory, 'some', 'subdirectory', 'here');
+        expect(() => folder.create()).toThrow();
+        expect(() => folder.create({ intermediates: false })).toThrow();
+        expect(folder.exists).toBe(false);
+      });
+
       it('Creates an empty file', () => {
         const file = new File(testDirectory + 'newFolder');
         file.create();
         expect(file.exists).toBe(true);
         expect(file.text()).toBe('');
       });
 
+      it('Throws an error if the file exists', () => {
+        const file = new File(testDirectory + 'newFolder');
```

Comment:
When are you going to stop doing this? 😛 
```suggestion
        const file = new File(testDirectory, 'newFolder');
```